src/content/en/2021/jamstack.md

```diff
@@ -205,9 +218,13 @@ On the other hand, Cloudflare, Akamai, AWS are used not only for pure serving as
   sql_file="core_web_vitals_distribution.sql"
 ) }}
 
-Next.js, as most popular by data we have, mostly served by Cloudflare, Vercel, and AWS. Most of Gatsby's sites use Netlify, AWS, and Cloudflare.
+{# TODO - this figure description should be described more #}
+
+Next.js, is the most popular, mostly served by Cloudflare, Vercel, and AWS. Most of Gatsby's sites use Netlify, AWS, and Cloudflare.
 Nuxt.js sites preferred to be served by Cloudflare, AWS, and Netlify. Hugo mostly uses Netlify, and Jekyll as no surprise mostly on GitHub.
 
+{# TODO - I did not understand the last sentence, is the comma used correctly? #}
```

Comment:
```suggestion
Next.js, is the most popular, mostly served by Cloudflare, Vercel, and AWS. Most of Gatsby's sites use Netlify, AWS, and Cloudflare. Nuxt.js sites preferred to be served by Cloudflare, AWS, and Netlify. Hugo mostly uses Netlify, and it's no surprised that Jekyll is used mostly on GitHub.
```

Why's it no surprise btw? Is that the only one really supported by GitHub? Did GitHub invent Jekyll? Or some other reason?